sdk/schemaregistry/schema-registry-avro/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.0.1-beta.1 (Unreleased)
```

Comment:
Did you mean 1.1.0-beta.1? Also does this have to be a beta? Are we expecting customer feedback on it?
```suggestion
## 1.1.0-beta.1 (Unreleased)
```

Reply:
Actually the client is a dep and it is in beta, so you're right this should be in beta as well.

Reply:
Unless we can depend on the GA version of the client which I think we can. Is there anything the avro serializer needs from the beta client?

Reply:
The serializer should work with the GA version of the client, I'll test it

Reply:
I'm currently updating this as a beta version. I'll test the edge cases to see if it's breaking anything for a GA version